app/controllers/pages_controller.rb

```diff
@@ -0,0 +1,21 @@
+class PagesController < ApplicationController
+  include HighVoltage::StaticPage
+  before_action :authenticate_user!
+
+  def show
+    render page_to_render
+  end
+
+  private
+
+  def page_to_render
+    case params[:id]
+    when 'instructions'
+      'instructions'
+    when 'ca_labs'
+      'ca_labs'
+    else
+      raise 'You have not defined the page yet'
+    end
+  end
```

Comment:
We definitely don't want to raise an error. 
I think you can safely remove the `show` method altogether and use the route setting from the gem docs and let the gem do the magic. ;-)